apis/camphor_tree_api.py

```diff
@@ -12,36 +12,40 @@ def send_satellite_message(email, info_level, message_body, server_option):
                                                message_to_encode=message_body)
     if server_option == 'Satsuki':
         rock_block_message.send_cloud_loop_message()
-    if server_option == 'Mei':
+        send_status = 'Send Success'
+    elif server_option == 'Mei':
```

Comment:
Have you considered putting multiple routes in your API to not need multiple modes in the one API call, e.g. call `/mei` or `/satsuki`, etc.?

Reply:
Ah, yeah.  I originally kept everything to one endpoint to avoid using any session cookies when putting the website behind a password :sweat_smile: 

PythonAnywhere comes with password-locked access as a platform option instead, so I'll look into pulling apart the functionality on another branch o/